.coderabbit.yaml

```
@@ -111,4 +111,6 @@ reviews:
     - "!**/dist/**"
 
 chat:
-  auto_reply: true 
\ No newline at end of file
+  auto_reply: true 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)